src/rules/noUnusedVariableRule.ts

```diff
@@ -88,7 +132,22 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
     }
 
     public visitNamespaceImport(node: ts.NamespaceImport) {
-        this.validateReferencesForVariable(node.name.text, node.name.getStart());
+        let importDecl = <ts.ImportDeclaration>node.parent.parent;
+        let modSpec = importDecl.moduleSpecifier.getText();
+        let match = modSpec.match(MODULE_SPECIFIER_MATCH);
```

Comment:
maybe add a comment explaining something like "match quoted import names and retrieve name without quotes"? Feel free to reword that, just that the intent of regex matches can be a little hazy w/o further explanation